code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -251,6 +251,9 @@
 		return occupant
 
 /obj/machinery/atmospherics/components/unary/cryo_cell/container_resist(mob/living/user)
+	if(user.next_move > world.time)
+		return
+	user.changeNext_move(CLICK_CD_RESIST)
 	to_chat(user, "<span class='notice'>You struggle inside the cryotube, kicking the release with your foot... (This will take around 30 seconds.)</span>")
 	audible_message("<span class='notice'>You hear a thump from [src].</span>")
 	if(do_after(user, 300))

```

Comment:
at the top of the proc, 
```
if(escape_in_progress) 
    to_chat(user, "<span class='notice'>You are already trying to exit (This will take around 30 seconds)</span>")
    return
```
above this line, ```escape_in_progress = TRUE```
at the bottom of the proc, ```escape_in_progress = FALSE```
and on the type, ```var/escape_in_progress = FALSE```